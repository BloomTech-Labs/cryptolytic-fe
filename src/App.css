.App {
  text-align: center;
  background: black;
}

/* .Main {
  display: flex;

} */

.dash {
  border-radius: 200px;
}

body {
  background: black;
}

/* .navHead {
  display: flex;
  justify-content: flex-end;
  flex-direction: column;
} */

/* .footer {
  display: flex;
  color: aliceblue;
  height: 10vh;
  justify-content: center;
  margin: 0 auto;
} */
