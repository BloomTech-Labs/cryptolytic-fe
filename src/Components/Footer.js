import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { Link, Grid } from "@material-ui/core";
import Typography from '@material-ui/core/Typography';

const useStyles = makeStyles(t => ({
	footer: {
		backgroundColor: "black",
		display: "flex",
		justifyContent: "flex-end",
		alignItems: "center",
		height: "10vh",
		width: "90%",
		padding: '60px 0px 225px 0px',

	},
	footerNavigation: {
		zIndex: 1,
		backgroundColor: "black",
		color: "#f0f8ff",
		display: "flex",
		alignItems: "center",
		width: "70%",
		height: "100%",
		paddingTop: 30,
		marginTop: 150
	},
	footerLinks: {
		fontFamily: 'Titillium Web, sans-serif',
		fontStyle: 'normal',
		fontWeight: 600,
		fontSize: '1.6rem',
		lineHeight: '49px',
		display: 'flex',
		alignItems: 'center',
		textAlign: 'center',
		margin: '5px',
		'&:hover': {
			textDecoration: 'none',
			color: 'gray'
		}
	},
	footerTitle: {
		fontFamily: 'Titillium Web, sans-serif',
		fontStyle: 'normal',
		fontWeight: 'bold',
		fontSize: '3rem',
		lineHeight: '76px',
		display: 'flex',
		alignItems: 'center',
		textAlign: 'center',
		letterSpacing: '0.03em'
	},
	footerBody: {
		fontFamily: 'Titillium Web, sans-serif',
		fontStyle: 'normal',
		fontWeight: 600,
		fontSize: '1.6rem',
		lineHeight: '49px',
		display: 'flex',
		alignItems: 'center',
		textAlign: 'center'
	},
	footerCopyright: {
		fontFamily: 'Titillium Web, sans-serif',
		fontStyle: 'normal',
		fontWeight: 'normal',
		fontSize: '1.4rem',
		lineHeight: '37px',
		display: 'flex',
		alignItems: 'center',
		textAlign: 'center',
		marginLeft: '1vw'
	}
}));

const Footer = () => {
	const classes = useStyles();
	return (
		<div className={classes.footer}>
			<nav className={classes.footerNavigation}>
				<Grid container spacing={0}>
					<Grid
						item
						sm={4}
						direction='column'
						container
						alignItems='flex-start'
					>
						<Typography className={classes.footerTitle}>Cryptolytic</Typography>
						<div style={{ margin: '30px 0px' }}>
							{['Contact Us', 'cryptolyticapp@gmail.com'].map(e =>
								<Typography className={classes.footerBody}>{e}</Typography>
							)}
						</div>
						<Typography className={classes.footerCopyright}>Copyright 2020 Cryptolytic</Typography>
					</Grid>
					<Grid item sm={4} container direction='column' alignItems='center'>
						{['Home', 'About', 'Services'].map(e =>
							<Link color='inherit' className={classes.footerLinks}>{e}</Link>
						)}
					</Grid>
					<Grid item sm={3} container direction='column' alignItems='center'>
						{['Legal', 'Privacy Policy', 'User Agreement', 'FAQ'].map(e =>
							<Link color='inherit' className={classes.footerLinks}>{e}</Link>
						)}
					</Grid>
				</Grid>
			</nav>
		</div>
	);
};

export default Footer;
