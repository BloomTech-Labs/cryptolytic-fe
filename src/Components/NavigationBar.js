import React from "react";
import { NavLink } from "react-router-dom";
import { makeStyles } from "@material-ui/core/styles";
import logo from "../images/logo.svg";
import home from "../images/home-logo.svg";
import arbitrage from "../images/arbitrage-logo.svg";
import trading from "../images/line-chart-logo.svg";
import candlestick from "../images/candlestick-icon2.png";
import api from "../images/cloud-upload-logo.svg";
import data from "../images/data-logo.svg";
import Link from "@material-ui/core/Link";

import AlertDialog from "./AlertDialog";

const useStyles = makeStyles(theme => ({
	NavBar: {
		width: "17vw",
		height: "100vh",
		backgroundColor: "black",
		boxShadow: "1px 0px 0px",
		borderRight: "1px solid #FFFFFF",
		position: "fixed",
		top: 0,
		left: 0,
		zIndex: 100,
		"@media(max-width: 1300px)": {
			width: "100vw",
			height: "10vh",
			position: "static"
		}
	},
	NavBarLogo: {
		width: "100%",
		height: "100px",
		display: "flex",
		justifyContent: "center",
		alignItems: "center",
		"@media(max-width: 1300px)": {
			display: "none"
		}
	},
	NavBarMenu: {
		paddingTop: "3vh",
		width: "100%",
		display: "flex",
		alignItems: "center",
		flexDirection: "column",
		color: "#FFFFFF",
		marginLeft: 15,
		"@media(max-width: 1300px)": {
			flexDirection: "row",
			padding: 0,
			margin: 0,
			height: "10vh"
		}
	},
	navBarMenuText: {
		width: "100%",
		textAlign: "left",
		fontWeight: 600,
		fontSize: "1.2rem",
		lineHeight: "33px",
		letterSpacing: "0.03em",
		display: "flex",
		textDecoration: "none",
		color: "#fff"
	},
	navBarMenuTextDisabled: {
		width: "100%",
		textAlign: "left",
		fontWeight: 600,
		fontSize: "1.2rem",
		lineHeight: "33px",
		letterSpacing: "0.03em",
		display: "flex",
		textDecoration: "none",
		color: "gray"
	},
	NavBarMenuContainer: {
		"@media(max-width: 1300px)": {
			display: "flex",
			width: "85%"
		},
		"&:hover": {
			backgroundColor: "rgba(35, 32, 44, 0.9)",
			width: "auto",
			marginLeft: 20,
			position: "relative"
		}
	},
	navBarMenuTextCursorActive: {
		backgroundColor: "#4EB9FF",
		width: 20,
		marginLeft: "-18vw",
		"@media(max-width: 1300px)": {
			width: 0,
			marginLeft: "0vw"
		}
	},
	navBarMenuButton: {
		visibility: "visible",
		display: "flex",
		marginBottom: "10px",
		alignItems: "center",
		width: "100%",
		height: "100%",
		position: "relative",
		transition: "0.2s ease",
		"@media(max-width: 1300px)": {
			marginBottom: "20px",
			"&:hover": {
				boxShadow: "1px 10px #4EB9FF"
			}
		}
	},
	navBarMenuIcon: {
		margin: "0px 10px",
		marginLeft: 50
	}
}));

function NavigationBar() {
	const classes = useStyles();
	return (
		<div className={classes.NavBar}>
			<div className={classes.NavMenuContainer}>
				<div className={classes.NavBarLogo}>
					<Link href='/'>
						<img src={logo} className={classes.Logo} alt='logo' />
					</Link>
				</div>
				<div className={classes.NavBarMenu}>
					{[
						{ name: "Home", logo: home, link: "home", status: "active" },
						{
							name: "Arbitrage",
							logo: arbitrage,
							link: "arbitrage-dashboard",
							status: "active"
						},
						{
							name: "Trading",
							logo: trading,
							link: "trading-dashboard",
							status: "active"
						},
						{
							name: "Charts",
							logo: candlestick,
							link: "exchange-charts",
							status: "active"
						},
						{ name: "API", logo: api, link: "api", status: "inactive" },
						{ name: "Data", logo: data, link: "data", status: "inactive" }
					].map(e =>
						e.status === "inactive" ? (
							<AlertDialog
								button={
									<NavLink
										activeClassName={classes.navBarMenuTextCursor}
										to={e.link}
										className={classes.navBarMenuTextDisabled}
									>
										<div className={classes.navBarMenuButton}>
											<img
												className={classes.navBarMenuIcon}
												style={{ margin: "0px 10px", marginLeft: 50 }}
												src={e.logo}
												alt={e.name}
											/>
											{e.name}
										</div>
									</NavLink>
								}
								title={"Coming Soon"}
								message={`We're still working on features included in the ${e.name} page. Come back soon!`}
							/>
						) : (
								<div style={{ all: "inherit" }}>
									<NavLink
										activeClassName={classes.navBarMenuTextCursorActive}
										to={e.link}
										className={classes.navBarMenuText}
									>
										<div className={classes.navBarMenuButton}>
											<img
												className={classes.navBarMenuIcon}
												src={e.logo}
												alt={e.name}
											/>
											{e.name}
										</div>
									</NavLink>
								</div>
							)
					)}
				</div>
			</div>
		</div>
	);
}

export default NavigationBar;
