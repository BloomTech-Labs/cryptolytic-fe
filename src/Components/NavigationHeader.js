import React, { useState } from "react";
import { makeStyles } from "@material-ui/core/styles";
import cogWheel from "../images/cogwheel.svg";
import Avatar from "@material-ui/core/Avatar";
import Menu from "@material-ui/core/Menu";
import MenuItem from "@material-ui/core/MenuItem";
import firebase from "../firebase";

const useStyles = makeStyles(theme => ({
	navHeader: {
		backgroundColor: "#000000",
		width: "100%",
		height: "10vh",
		display: "flex",
		justifyContent: "space-between",
		"@media(max-width: 1300px)": {
			display: "flex",
			justifyContent: "space-around",
			width: "15%"
		}
	},
	title: {
		width: "20vw",
		height: "10vh",
		marginLeft: "18vw",
		display: "flex",
		justifyContent: "center",
		alignItems: "center",
		color: "#FFFFFF",
		fontWeight: 300,
		fontSize: "30px",
		lineHeight: "46px",
		letterSpacing: "0.03em",
		marginBottom: "30vw",
		"@media(max-width: 1300px)": {
			display: "none"
		}
	},
	cogWheel: {
		width: "10%",
		height: "10vh",
		display: "flex",
		justifyContent: "space-evenly",
		alignItems: "center",
		"@media(max-width: 1300px)": {
			width: "100%"
		}
	}
}));

function NavigationHeader({ history }) {
	const [anchorEl, setAnchorEl] = useState(null);
	const classes = useStyles();

	const handleClick = event => {
		setAnchorEl(event.currentTarget);
	};

	const handleClose = () => {
		setAnchorEl(null);
	};

	const handleLogout = () => {
		localStorage.removeItem("firebase_jwt");
		firebase.auth().signOut();
		history.push("/");
	};

	return (
		<div className={classes.navHeader}>
			<div className={classes.title}></div>
			<div className={classes.cogWheel}>
				<img src={cogWheel} />
				<div>
					<Avatar
						aria-controls='profile-menu'
						aria-haspopup='true'
						onClick={handleClick}
						style={{ cursor: "pointer" }}
					/>
					<Menu
						id='profile-menu'
						anchorEl={anchorEl}
						keepMounted
						open={Boolean(anchorEl)}
						onClose={handleClose}
					>
						<MenuItem onClick={handleLogout}>Logout</MenuItem>
					</Menu>
				</div>
			</div>
		</div>
	);
}

export default NavigationHeader;
