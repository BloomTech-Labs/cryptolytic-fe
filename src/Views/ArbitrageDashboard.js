import React from "react";
import DashboardTable from "../Components/DashboardTable";

import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles({
	mainHeader: {
		background:
			"-webkit-linear-gradient(92.5deg, #4EB9FF 19.25%, #53CFD7 45.13%, #5DDCB7 67.95%, #62E3AB 82.93%)",
		WebkitBackgroundClip: "text",
		WebkitTextFillColor: "transparent",
		position: "relative",
		width: "180px",
		height: "53px",
		left: "24.3%",
		fontWeight: "180px",
		fontSize: "2.2em",
		lineHeight: "3em",
		display: "flex",
		alignItems: "center",
		letterSpacing: "0.03em"
	},
	secondaryHeader: {
		position: "relative",
		left: "21.7%",
		color: "white",
		width: "800px",
		height: "34px",
		fontWeight: "600",
		fontSize: "1.2em",
		lineHeight: "1.2em"
	}
});

let deletedColumns = ['_id', '__v', 'trade_time'];
let orderedColumns = ['date', 'time', 'buy_exchange', 'sell_exchange', 'trading_pair', 'price_difference', 'arbitrage_percentage'].reverse()
let tradingPairArray = [4]
let exchangeArray = [2, 3]
let percentageArray = [6]
let fiatArray = []
let cryptoArray = [5]
let predictionsArray = []
let largeNumberArray = []
let tickerArray = []
let endpointUrl = "https://cryptolytic-starter.herokuapp.com/arbitrage"
let arbitrageProps = { deletedColumns, orderedColumns, tradingPairArray, exchangeArray, percentageArray, fiatArray, cryptoArray, endpointUrl, largeNumberArray, tickerArray, predictionsArray }


const ArbitrageDashboard = () => {
	const classes = useStyles();

	return (
		<div>
			<h1 className={classes.mainHeader}>Arbitrage</h1>
			<h3 className={classes.secondaryHeader}>
				Buy at the first exchange and sell at the second 30 minutes later to
        make a profit!
      </h3>
			<DashboardTable {...arbitrageProps} />
		</div>
	);
};

export default ArbitrageDashboard;
