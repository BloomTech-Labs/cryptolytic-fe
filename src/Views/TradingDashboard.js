import React from "react";
import DashboardTable from "../Components/DashboardTable";

import { makeStyles } from "@material-ui/core/styles";

const useStyles = makeStyles({
	mainHeader: {
		background:
			"-webkit-linear-gradient(92.5deg, #4EB9FF 19.25%, #53CFD7 45.13%, #5DDCB7 67.95%, #62E3AB 82.93%)",
		WebkitBackgroundClip: "text",
		WebkitTextFillColor: "transparent",
		position: "relative",
		width: "140px",
		height: "53px",
		left: "24.3%",
		fontWeight: "600",
		fontSize: "2.2em",
		lineHeight: "3em",
		display: "flex",
		alignItems: "center",
		letterSpacing: "0.03em"
	},
	secondaryHeader: {
		position: "relative",
		left: "23.7%",
		color: "white",
		width: "800px",
		height: "34px",
		fontWeight: "600",
		fontSize: "1.2em",
		lineHeight: "1.2em"
	}
});

	///// Working Trading configuration
	let deletedColumns = ['model_type', 'period', 'timestamp'];
	let orderedColumns = [].reverse()
	let tradingPairArray = [3]
	let exchangeArray = [2]
	let percentageArray = []
	let fiatArray = []
	let cryptoArray = []
	let largeNumberArray = []
	let tickerArray = []
	let predictionsArray = [4]
	let endpointUrl = "https://cryptolytics-sample-ds.herokuapp.com/liveDataRoute/getTradePredictionsClean"
	let tradeProps = { deletedColumns, orderedColumns, tradingPairArray, exchangeArray, percentageArray, fiatArray, cryptoArray, endpointUrl, largeNumberArray, tickerArray, predictionsArray }

const TradingDashboard = () => {
	const classes = useStyles();

	return (
		<>

			<h1 className={classes.mainHeader}>Trading</h1>
			<h3 className={classes.secondaryHeader}>
				Price movement predictions for cryptocurrency trading pairs based on
        technical analysis.
      		</h3>
			<DashboardTable  {...tradeProps} />
		</>
	);
};

export default TradingDashboard;
