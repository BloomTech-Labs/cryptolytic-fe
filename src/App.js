import React from "react";
import { Route, BrowserRouter as Router } from "react-router-dom";
import { makeStyles } from "@material-ui/core/styles";
import TradingDashboard from "./Views/TradingDashboard";
import ArbitrageDashboard from "./Views/ArbitrageDashboard";
import "./App.css";
import NavigationHeader from "./Components/NavigationHeader";
import NavigationBar from "./Components/NavigationBar";
import Footer from "./Components/Footer";
import Charts from "./Views/Charts";
import MarketIndex from "./Components/Market Index/MarketIndex";
import Login from "./Components/Registration/Login";
import SignUp from "./Components/Registration/SignUp";
import IntroIndex from "./Components/Marketing Page/IntroIndex";
import Services from "./Components/Marketing Page/Services";
import TeamPage from "./Components/TeamPage/TeamPage";
const useStyles = makeStyles(theme => ({
	navContainer: {
		"@media(max-width: 1300px)": {
			display: "flex",
			flexDirection: "row-reverse"
		}
	}
}));
function App() {
	const classes = useStyles();
	return (
		<div className='App'>
			<Router>
				<Route exact path='/login' render={props => <Login {...props} />} />
				<Route path='/signup' render={props => <SignUp {...props} />} />
				<Route exact path='/' component={IntroIndex} />
				<Route exact path='/services' component={Services} />
				<Route exact path='/team' component={TeamPage} />
				<Route
					path='/main'
					render={({ match: { url }, history }) => (
						<>
							<div className='navHeader'>
								{!window.location.href.includes("intro") ? (
									<div className={classes.navContainer}>
										<NavigationHeader />
										<NavigationBar />
									</div>
								) : (
										<></>
									)}
							</div>
							<Route path={`${url}/home`} component={MarketIndex} />
							<Route
								path={`${url}/trading-dashboard`}
								component={TradingDashboard}
							/>
							<Route
								path={`${url}/arbitrage-dashboard`}
								component={ArbitrageDashboard}
							/>
							<Route path={`${url}/exchange-charts`} component={Charts} />
							<Footer />
						</>
					)}
				/>
			</Router>
		</div>
	);
}
export default App;
