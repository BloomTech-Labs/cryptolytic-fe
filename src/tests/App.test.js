import React from "react";
///
import App from "../App";
import TradingDashboard from "../Views/TradingDashboard";
import ArbitrageDashboard from "../Views/ArbitrageDashboard";
import NavigationHeader from "../Components/NavigationHeader";
import NavigationBar from "../Components/NavigationBar";
import Footer from "../Components/Footer";
///
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import renderer from "react-test-renderer";

test("Dashboard components initialized without crashing!", () => {
	render(<TradingDashboard />);
	render(<ArbitrageDashboard />);
	render(<NavigationHeader />);
	render(
		<MemoryRouter>
			<NavigationBar />
		</MemoryRouter>
	);
	render(<Footer />);

});
test("<App /> should rende r", () => {
	render(
		<MemoryRouter>
			<App />
		</MemoryRouter>
	);
});

describe('<App /> matches verified SnapShot', () => {
	it('should match snapshot', () => {
		const tree = renderer.create(
			<MemoryRouter>
				<App />
			</MemoryRouter>
		).toJSON();
		expect(tree).toMatchSnapshot();
	});
});