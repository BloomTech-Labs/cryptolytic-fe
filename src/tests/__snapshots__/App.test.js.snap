// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches verified SnapShot should match snapshot 1`] = `
<div
  className="App"
>
  <div
    style={
      Object {
        "fontFamily": "Titillium Web",
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <div
      className="makeStyles-headerContainer-345"
    >
      <div
        className="makeStyles-headerLeft-346"
      >
        <div
          className="makeStyles-headerLogoContainer-347"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-headerLogoLink-348 MuiTypography-colorPrimary"
            href="/"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            <img
              className="makeStyles-headerLogo-349"
              src="logo.svg"
            />
            <p>
              Cryptolytic
            </p>
          </a>
        </div>
      </div>
      <div
        className="makeStyles-headerRightContainer-350"
      >
        <div
          className="makeStyles-headerMenuContainer-351"
        >
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-headerMenuLink-352 MuiTypography-colorPrimary"
            href="/services"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Services
          </a>
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-headerMenuLink-352 MuiTypography-colorPrimary"
            href="/team"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Team
          </a>
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-headerMenuLink-352 MuiTypography-colorPrimary"
            href="/login"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Login
          </a>
          <a
            className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-headerMenuLink-352 MuiTypography-colorPrimary"
            href="/signup"
            onBlur={[Function]}
            onFocus={[Function]}
          >
            Sign Up
          </a>
        </div>
      </div>
    </div>
    <div
      className="makeStyles-heroContainer-390"
    >
      <div
        className="makeStyles-heroText-391"
      >
        <h1
          className="makeStyles-heroTitle-392"
        >
          Smarter trading
        </h1>
        <h2>
          Discover arbitrage opportunities in the cryptocurrency markets before they happen and make profitable trades
        </h2>
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-heroLink-393 MuiTypography-colorPrimary"
          href="/main/home"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          Get Started
        </a>
      </div>
      <div
        className="makeStyles-heroSpace-394"
      />
    </div>
    <div
      className="makeStyles-homeFeatureContainer-395"
    >
      <div
        style={
          Object {
            "paddingTop": "30vh",
          }
        }
      >
        <div
          className="makeStyles-homeFeatureRow-396"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <div
            className="makeStyles-homeFeatureRowImage-397"
            style={
              Object {
                "justifyContent": "flex-end",
              }
            }
          >
            <img
              src="artboard_1 (1) 2.png"
            />
          </div>
          <div
            className="makeStyles-homeFeatureRowText-398"
            style={
              Object {
                "justifyContent": "flex-start",
                "textAlign": "left",
              }
            }
          >
            <div>
              <h2
                className="makeStyles-homeFeatureRowTitle-399"
              >
                Arbitrage Forecasts
              </h2>
              <p>
                Finding opportunities to profit in exciting cryptocurrency markets doesn’t have to be difficult. Introducing Cryptolytic’s Arbitrage trading strategy, which provides users with in-advance, lower risk, profitable trade suggestions.
              </p>
            </div>
          </div>
        </div>
        <div
          className="makeStyles-homeFeatureRow-396"
          style={
            Object {
              "flexDirection": "row-reverse",
            }
          }
        >
          <div
            className="makeStyles-homeFeatureRowImage-397"
            style={
              Object {
                "justifyContent": "flex-start",
              }
            }
          >
            <img
              src="arb graphic.png"
            />
          </div>
          <div
            className="makeStyles-homeFeatureRowText-398"
            style={
              Object {
                "justifyContent": "flex-end",
                "textAlign": "right",
              }
            }
          >
            <div>
              <h2
                className="makeStyles-homeFeatureRowTitle-399"
              >
                What is arbitrage?
              </h2>
              <p>
                Arbitrage is the practice of taking advantage of price discrepancies in different markets. When an arbitrage opportunity exists, you could purchase an item from the lower priced market, and sell in the higher priced market simultaneously to make a profit.  Arbitrage commonly occurs in financial markets and is a completely legal trading strategy for cryptocurrencies.
              </p>
            </div>
          </div>
        </div>
        <div
          className="makeStyles-homeFeatureRow-396"
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <div
            className="makeStyles-homeFeatureRowImage-397"
            style={
              Object {
                "justifyContent": "flex-end",
              }
            }
          >
            <img
              src="Crypto_compute (1) 1.png"
            />
          </div>
          <div
            className="makeStyles-homeFeatureRowText-398"
            style={
              Object {
                "justifyContent": "flex-start",
                "textAlign": "left",
              }
            }
          >
            <div>
              <h2
                className="makeStyles-homeFeatureRowTitle-399"
              >
                Trade Recommendations
              </h2>
              <p>
                Cryptolytic’s Trade Recommender provides users with advance insights into market movements based off of millions of data points and an assortment of technical indicators. Take more control of your investments today by incorporating Cryptolytic’s machine learning algorithms into your strategy.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="makeStyles-homeFeatureLastRow-400"
      >
        <div
          className="makeStyles-homeFeatureLastRowText-401"
        >
          <div>
            <h2
              className="makeStyles-homeFeatureLastRowTitle-402"
            >
              Automate your trading process
            </h2>
            <p
              className="makeStyles-homeFeatureLastRowSubtitle-403"
            >
              The cryptocurrency market doesn’t sleep and neither should your money! Connect the Cryptolytic API to a trading bot and act on trading and arbitrage opportunities automatically 24/7!
            </p>
          </div>
        </div>
        <div
          className="makeStyles-homeFeatureLastRowImage-404"
        >
          <img
            src="cryptocurrency-hero 1.png"
          />
        </div>
      </div>
    </div>
    <div
      className="makeStyles-accountFeaturesContainer-405"
    >
      <h2>
        Account Features
      </h2>
      <div
        className="makeStyles-accountFeaturesRow-406"
      >
        <div
          className="makeStyles-accountFeaturesCard-407"
        >
          <div
            className="makeStyles-accountFeaturesText-408"
          >
            <h3
              style={
                Object {
                  "color": "#4EB9FF",
                }
              }
            >
              Arbitrage Dashboard
            </h3>
            <p>
              User friendly display for tracking arbitrage trading opportunities across exchanges
            </p>
          </div>
        </div>
        <div
          className="makeStyles-accountFeaturesCard-407"
        >
          <div
            className="makeStyles-accountFeaturesText-408"
          >
            <h3
              style={
                Object {
                  "color": "#4FC9E5",
                }
              }
            >
              Trade Recommender Dashboard
            </h3>
            <p>
              Simple display to view predictions on market movement
            </p>
          </div>
        </div>
        <div
          className="makeStyles-accountFeaturesCard-407"
        >
          <div
            className="makeStyles-accountFeaturesText-408"
          >
            <h3
              style={
                Object {
                  "color": "#57D5C7",
                }
              }
            >
              API Access
            </h3>
            <p>
              Arbitrage and trading APIs for up-to-date predictions in the market
            </p>
          </div>
        </div>
        <div
          className="makeStyles-accountFeaturesCard-407"
        >
          <div
            className="makeStyles-accountFeaturesText-408"
          >
            <h3
              style={
                Object {
                  "color": "#63E4A7",
                }
              }
            >
              Open Source Dataset
            </h3>
            <p>
              Access to current datasets for all supported exchanges
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="makeStyles-statsContainer-409"
    >
      <div
        className="makeStyles-statsTitle-410"
      >
        <h2>
          Powered by Machine Learning
        </h2>
      </div>
      <div
        className="makeStyles-statsTitleBackground-411"
      />
      <div
        className="makeStyles-statsRowContainer-412"
      >
        <div
          className="makeStyles-statsLeftContainer-413"
        >
          <div
            className="makeStyles-statsLeftText-414"
          >
            <h3>
              18M+
            </h3>
            <p
              className="makeStyles-statsLeftSubtitle-415"
            >
              Data points analyzed
            </p>
          </div>
        </div>
        <div
          className="makeStyles-statsMiddleContainer-416"
        >
          <div>
            <h3>
              30
            </h3>
            <p
              className="makeStyles-statsMiddleSubtitle-417"
            >
              predictive models
            </p>
          </div>
        </div>
        <div
          className="makeStyles-statsRightContainer-418"
        >
          <div
            className="makeStyles-statsRightText-419"
          >
            <p
              className="makeStyles-statsRightTitle-420"
            >
              trained on
            </p>
            <h3>
              20K+
            </h3>
            <p
              className="makeStyles-statsRightSubtitle-421"
            >
              historical arbitrages
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="makeStyles-partnersContainer-422"
    >
      <h2
        className="makeStyles-partnersTitle-423"
      >
        Supported Exchanges
      </h2>
      <div
        className="makeStyles-partnersTopRow-424"
      >
        <img
          src="bitfinex_logo 1.svg"
        />
        <img
          src="coinbase_logo.svg"
        />
        <img
          src="hitbtc_logo.svg"
        />
        <img
          src="kraken_logo.svg"
        />
        <img
          src="gemini_logo.svg"
        />
      </div>
      <div
        className="makeStyles-partnersBottomRow-425"
      >
        <img
          src="btc logo.png"
        />
        <img
          src="ltc logo.png"
        />
        <img
          src="dash logo.png"
        />
        <img
          src="eos logo.png"
        />
        <img
          src="ripple logo.png"
        />
      </div>
    </div>
    <div
      className="makeStyles-ctaContainer-426"
    >
      <h2
        className="makeStyles-ctaTitle-427"
      >
        Supercharge your trading with Cryptolytic
      </h2>
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-ctaButton-428 MuiTypography-colorPrimary"
        href="/signup"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        Sign Up for a free account
      </a>
      <p
        className="makeStyles-ctaDisclaimer-429"
      >
        Cryptolytic does not offer financial advice. Trade at your own risk.
      </p>
    </div>
  </div>
</div>
`;
